apps/mark-scan/backend/src/pat-input/logging.c

```diff
@@ -0,0 +1,51 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <errno.h>
+
+#include "logging.h"
+
+void print_log_common(char *event_id, char *event_type, char *message, char *operation, char *disposition)
+{
+  printf(
+      "{\"eventId\": \"%s\", \"source\": \"vx-mark-scan-pat-input-daemon\", \"eventType\": \"%s\", \"user\": \"system\", \"message\": \"%s\", \"operation\": \"%s\", \"disposition\": \"%s\"}\n",
+      event_id,
+      event_type,
+      message,
+      operation,
+      disposition);
+}
```

Comment:
I wasn't sure if going from 0 to 1 C deps (for `json-c` or similar) would introduce build headaches. Since the logging needs are somewhat simple this felt acceptable, but open to feedback or ideas.